pkg/scheduler/framework/runtime/framework.go

```diff
@@ -561,33 +561,47 @@ func (f *frameworkImpl) QueueSortFunc() framework.LessFunc {
 // *Status and its code is set to non-success if any of the plugins returns
 // anything but Success. If a non-success status is returned, then the scheduling
 // cycle is aborted.
-func (f *frameworkImpl) RunPreFilterPlugins(ctx context.Context, state *framework.CycleState, pod *v1.Pod) (status *framework.Status) {
+func (f *frameworkImpl) RunPreFilterPlugins(ctx context.Context, state *framework.CycleState, pod *v1.Pod) (_ *framework.PreFilterResult, status *framework.Status) {
 	startTime := time.Now()
 	defer func() {
 		metrics.FrameworkExtensionPointDuration.WithLabelValues(preFilter, status.Code().String(), f.profileName).Observe(metrics.SinceInSeconds(startTime))
 	}()
+	var result *framework.PreFilterResult
+	var pluginWithNodes string
+	errMessage := func(pl framework.Plugin) string {
+		if pluginWithNodes == "" {
+			return fmt.Sprintf("node(s) didn't match %s", pl.Name())
+		}
+		return fmt.Sprintf("node(s) didn't match both %s and %s", pluginWithNodes, pl.Name())
```

Comment:
Do we need to distinguish `pluginWithNodes` and `pl`? `pl` actually may not be the _real_ smoking guy to blame for. Just returning a msg showing all `pluginWithNodes` may make more sense.

Note that `pluginWithNodes` can be more than one plugin.

Reply:
updated, it was just an artifact of previous logic I didn't cleanup